ScnSoft-Education/LeetCodeTraining/RomanToInteger-13/RomanToIntegerSolution.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace LeetCodeTraining.RomanToInteger
+{
+    //https://leetcode.com/problems/roman-to-integer/
+    public class RomanToIntegerSolution
+    {
+        public static int RomanToInt(string s)
+        {
+            Dictionary<char, int> romanMap = new Dictionary<char, int>() 
+            {
+                {'I', 1}, {'V', 5}, {'X', 10}, {'L', 50},
+                {'C', 100}, {'D', 500}, {'M', 1000}
+            };
+
+            int result = 0;
```

Comment:
Please remove extra line before and add line after this line to add vertical alignment for better code reading. Then please complete the PR.